docs/fips.md

```diff
@@ -0,0 +1,195 @@
+# BoringCrypto and FIPS 140-3 compliance
+
+## Introduction
+
+Sumo OT distro can be built to use BoringCrypto instead of Go standard library crypto implementations.
+This document explains how to obtain FIPS-capable binaries and how to run them in FIPS-approved mode.
+
+## Obtaining the binaries
+
+For general installation instructions, see [here](installation.md).
+
+### Using the install script
+
+Pass the `--fips` flag to the [script](./installation.md#installation-using-script). If you try to do this on an unsupported platform,
+the script will error.
+
+### Get the binaries directly
+
+The binaries can be downloaded from OT release pages. They are distinguished from others by containing the
+`-fips-` infix. More specifically, the binary name follow the format:
+
+```text
+otelcol-sumo-${VERSION}-fips-${OS}-${ARCH}
+```
+
+For example: <https://github.com/SumoLogic/sumologic-otel-collector/releases/download/v0.69.0-sumo-0/otelcol-sumo-0.69.0-sumo-0-fips-linux_amd64>
+
+### Docker images
+
+Docker images containing the binaries have their tag suffixed with `-fips`. For example, where the normal image for
+version `0.69.0-sumo-0` is:
+
+```text
+docker pull public.ecr.aws/sumologic/sumologic-otel-collector:0.69.0-sumo-0
+```
+
+The FIPS-approved version would be:
+
+```text
+docker pull public.ecr.aws/sumologic/sumologic-otel-collector:0.69.0-sumo-0-fips
+```
+
+> **Note**
+> Currently FIPS-compliant builds are only available for Linux on amd64.
+
+## Running in FIPS-approved mode
+
+The FIPS-compliant build statically enforces use of [FIPS 140-3][fips] compliant TLS settings, and doesn't require any
```

Comment:
This is confusing, as it conflates two different things: FIPS 140 validation of the BoringCrypto module, and settings that ensure that TLS 1.2 is used only with cipher suites that use approved primitives. (FIPS 140 does not regulate TLS.)

Maybe say something like "The FIPS-compliant build statically enforces the use of FIPS-approved algorithms in TLS cipher suites, and..." ?